src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent.mdx

```diff
@@ -59,21 +59,35 @@ To install the Node.js agent:
 
 1. Create a New Relic account. Don't have one? [Sign up for free!](https://newrelic.com/signup) No credit card required.
 2. Ensure you meet the [system requirements](/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent). In particular, make sure you use a supported Node.js version.
-3. Use the command `npm install newrelic --save` for each application you want to monitor.
+3. Use the command `npm install newrelic` for each application you want to monitor.
+  <Callout variant="important">
```

Comment:
It's also worth noting we do not bundle apollo plugin so you may want to call that out?

Reply:
Added in apollo server plugin, thanks Bob.